edge_server/chronos/app.py

```diff
@@ -216,6 +216,7 @@ async def get_device_state(
 @with_circuit_breaker
 @with_rate_limit
 async def update_device_state(data: DeviceModel):
+    raise ValueError("This endpoint is broken")
```

Comment:
this is a toy example of a relay switch not working. I would like to make sure the seasonal mode switch fails with this change i.e. the winter/summer valve needs to be toggled in order to switch modes on the interface